docs/cmdline-opts/mptcp.md

```diff
@@ -0,0 +1,38 @@
+---
+c: Copyright (C) Dorian Craps, <dorian.craps@student.vinci.be>
+SPDX-License-Identifier: curl
+Long: mptcp
+Added: 8.8.0
+Help: Enable Multipath TCP (MPTCP)
+Category: connection
+Multi: boolean
+See-also:
+  - tcp-fastopen
+Example:
+  - --mptcp $URL
+---
+
+# `--mptcp`
+
+Enables the use of Multipath TCP (MPTCP) for connections. MPTCP is an extension
+to the standard TCP that allows multiple TCP streams over different network
+paths between the same source and destination. This can enhance bandwidth and
+improve reliability by using multiple paths simultaneously.
+
+MPTCP is beneficial in networks where multiple paths exist between clients and
+servers, such as mobile networks where a device may switch between WiFi and
+cellular data or in wired networks with multiple Internet Service Providers.
+
+## Usage
+
+To use MPTCP for your connections, add the `--mptcp` option when using `curl`:
+
+## Requirements
+
+- This feature is currently only supported on Linux starting from kernel 5.6.
+- The server you are connecting to must also support MPTCP.
+  If not, the connection falls back to TCP..
```

Comment:
there is an extra dot `.` at the end here.

While at it, maybe good to add "seamlessly" and this line could be partially moved to the previous one, as long as you keep the 80 char limit:

```
- The server you are connecting to must also support MPTCP. If not, the
  connection will seamlessly fall back to TCP.
```